eclair-core/src/main/scala/fr/acinq/eclair/channel/publish/ReplaceableTxPublisher.scala

```diff
@@ -347,5 +374,10 @@ private class ReplaceableTxPublisher(nodeParams: NodeParams,
     }
   }
 
+  /** If we don't have a lot of safe utxos left, we will use an aggressive feerate to ensure our utxos aren't locked for too long. */
+  private def hasEnoughSafeUtxos(threshold: Int): Future[Boolean] = {
+    bitcoinClient.listUnspent().map(_.count(utxo => utxo.safe && utxo.amount >= 10_000.sat) >= threshold)
```

Comment:
We could have made the threshold value configurable, but this is short term.

Reply:
Yes, I hesitated but I don't think it's worth it, unless we hit a roadblock on the future use of unsafe utxos.